src/api/docs/content/specs/localdns.yaml

```diff
@@ -0,0 +1,199 @@
+openapi: 3.0.2
+components:
+  paths:
+    record_type:
+      parameters:
+        - $ref: 'localdns.yaml#/components/parameters/record_type'
+      get:
+        summary: Get local DNS records
+        tags:
+          - "Local DNS Records"
+        security: []
```

Comment:
Why are you defining that `security` is not required?

![image](https://github.com/user-attachments/assets/4b8b86cc-7542-4b12-a64d-7b98369fe72a)


This is definitely wrong. Just leave this away and the endpoint should correctly receive standard security restrictions:

```suggestion
```